### ISSUE TITLE
Cancellation of Pending Transaction

### ISSUE THREAD

**Issue by xBalbinus** [filecoin-project/fil-b#42]
**Description:**
Is it possible to cancel a pending transaction, eg a `InvokeContract` call?

Motivation: Currently, there's one message on https://filfox.info/en/address/f410fko6732us66tav34cekesnubipaayvs2o4pv46mi that has been pending for 8h. This is due to the rise in gas prices (https://filfox.info/en/stats/gas). We want to recover our system and get calls to work again. Our hope is that we can cancel such a transaction, and resubmit it with more gas. Would something like this help?

**Source:**
https://filecoinproject.slack.com/archives/CRK2LKYHW/p1722432098249869?thread_ts=1722432098.249869&cid=CRK2LKYHW

**Comment by longfeiWan9** [comment]
- Ribasushi [slack message](https://filecoinproject.slack.com/archives/C0179RNEMU4/p1607269156412600?thread_ts=1606987775.021900&cid=C0179RNEMU4) explaining how to cancel a message stuck in the mpool. 
- There is a [lotus doc](https://lotus.filecoin.io/kb/update-msg-gas-fee/) explaining how to increase gas fee for the pending transaction using Lotus cli.
@xBalbinus maybe we can write a tutorial on how to cancel or expedite the pending transactions? Usling Lotus CLI or API calls, or event with other tools if possible ?

**Comment by xBalbinus** [comment]
Yes, we should write some documentation on this. Normally though, both hardhat and foundry innately have capabilities for you to cancel / skip existing transactions, so this might end up even as a P2.

**Comment by trruckerfling** [comment]
> tutorial on how to cancel or expedite the pending transactions

There are 3 paths: 
1. This assumes Lotus CLI isn't readily available to general dev. Possible to do via ethers.js, to reconstruct pending transaction with updated gas fee (this is a common fix for evm devs, but filecoin devs might be unfamiliar)
2. For Lotus CLI, there's a docs page for this existing. No action needed. 
3. Another solution for txns initialized on metamask - there is a cancel feature.

End result should be a fix on [forum](https://fvm-forum.filecoin.io/) and amplified to builders
